src/app/shared/models/studio-configuration.ts

```diff
@@ -1,5 +1,6 @@
 interface BlueprintConfiguration {
   ServerPostrollDuration: number
+  JingleFolder: string
```

Comment:
These should ideally be camelCased. 
The reason they aren't is because that is how Core saves them and because of JavaScript we just get all attributes of an object even though sofieServer isn't specifying these attributes.

It would also require a backend change, so I would leave this as is for now. It's mainly an FYI :)